packages/teleport/src/services/apps/makeApps.ts

```diff
@@ -36,7 +36,7 @@ export default function makeApp(json): App {
     name,
     uri,
     publicAddr,
-    labels,
+    tags: labels.map(label => `${label.name}: ${label.value}`),
```

Comment:
no need to you lodash here:

```tsx
export default function makeApp(json: any): App {
  json = json || {};
  const { name, uri, publicAddr, labels, clusterId, fqdn } = json;
  const id = `${clusterId}-${name}-${publicAddr}`;
  const launchUrl = cfg.getAppLauncherRoute({ fqdn, clusterId, publicAddr });
  const tags = !Array.isArray(labels)
    ? []
    : labels.map(label => `${label.name}: ${label.value}`);

  return {
    id,
    name,
    uri,
    publicAddr,
    tags,
    clusterId,
    fqdn,
    launchUrl,
  };
}
```

Notice an extra check for nullable labels, this is because our backend does not guarantee that there always be an empty array.

Reply:
@rudream could you also fix floating "Play" icon?

![image](https://user-images.githubusercontent.com/7816406/123012782-e1ae9b00-d390-11eb-9485-347aed69d07f.png)

Reply:
+ Lets also make the row clickable as this is the primary action for applications. Reaching out to the "Open" button on wide screens is painful. As for "copy&past" we can detect if mouse button is pressed to ignore the click.

Reply:
@rudream lets finish with this PR by addressing the rest of the comments (like using the colors from palette)